scripts/valhalla_build_config

```diff
@@ -12,6 +12,7 @@ class optional:
 config = {
   'mjolnir': {
     'max_cache_size': 1000000000,
+    'id_table_size': 6800000000,
```

Comment:
I think I've seen OSM node Ids greater than 7000000000, may want to increase this default value

Reply:
ty and updated.

Reply:
FYI, there's a website that tracks the largest seen node ID in OSM:

http://textual.ru/64/

current value (2020-05-04) is 7,483,368,971